test/e2e/e2e_test.go

```diff
@@ -37,32 +42,35 @@ var (
 	communityOperatorsImage = ""
 )
 
-func TestConvert(t *testing.T) {
+func TestEndToEnd(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Convert Suite")
+	RunSpecs(t, "End-to-end")
 }
 
+var deprovision func() = func() {}
```

Comment:
Is there a reason `provision` is not capitalized?

Reply:
As in "deProvision"? I've only ever seen "deprovision" used a single word stemming from "provision" with the [de-](https://www.dictionary.com/browse/de-) prefix indicating reversal/negation, although "deprovision" doesn't appear in my dictionary.

Reply:
Trying to understanding this line, what does it actually mean in go?

Reply:
`deprovision` is a function set to be an empty function here. It gets assigned a different function based on the provisioner you choose.

It's defined here so that it's in scope for both `BeforeSuite` (where it gets defined) and `AfterSuite` (where it gets called).

It's an empty function instead of nil so that we don't have to bother checking nil at call sites.